Jenkinsfile-Release

```diff
@@ -17,23 +17,8 @@ final jira = [
     credentialId : 'jenkins-jira', autoRelease: true, failOnError: true
 ]
 
-final jiraVersionMappings = [
-  'nexus-iq-server': 'helm-iq',
-  'nexus-repository-manager': 'helm-nxrm',
-]
-
-final chartLocation = [
-  'nexus-iq-server': 'nexus-iq',
-  'nexus-repository-manager': 'nexus-repository-manager',
-]
-
 properties([
   parameters([
-    choice(
-      choices: ['', 'nexus-iq-server', 'nexus-repository-manager'],
-      name: 'chart',
-      description: 'Chart to deploy.',
-    ),
```

Comment:
without this choice, how do I tell the build that I want it to release nexus-iq chart (build, test, index, commit) vs releasing nexus-repository-manager (download, index, commit)?

Reply:
it needs to know and only do one of the other, since the appVersions and chartVersions are different between nxrm and iq charts.

Reply:
IQ team needs to be able to come to this build, choose "nexus-iq-server" and give it the appVersion like "1.144.0" to tell it to do that update and the rest of that build.
Repo team needs to be able to choose "nexus-repository-manager" and give it a chartVersion like "41.1.0" to have it do this new process.

Reply:
With this code, it is the only option is 'nexus-iq-server' hence the choice is redundant. The nexus-repository-manager can be released using the corresponding job from 'nxrm3-helm-repository'.

Reply:
Line 34-35 below seems to be using `params.appVersion` to download a version of the NXRM charts. Won't that fail when given version strings from IQ?

Maybe you could step back and explain to me how this Jenkinsfile is used to service IQ and NXRM.

Reply:
The code was reworked in order to keep the choice section. So, when IQ selected - all existed workflow will be evaluated but when nxrm selected then only artifacts will be downloaded and pushed into the repository.
The 'params.chartVersion' will be used for downloading artefacts